staging/src/k8s.io/apiextensions-apiserver/test/integration/fixtures/resources.go

```diff
@@ -143,6 +143,23 @@ func NewMultipleVersionNoxuCRD(scope apiextensionsv1beta1.ResourceScope) *apiext
 	}
 }
 
+// SetWebhookConverter configures the provided crd to use the provided webhook config for the supportedVersions.
+func SetWebhookConverter(crd *apiextensionsv1beta1.CustomResourceDefinition, webhookConfig *apiextensionsv1beta1.WebhookClientConfig, supportedVersions ...string) *apiextensionsv1beta1.CustomResourceDefinition {
```

Comment:
does this really need a helper? It just take the arrguments and puts them 1:1 into the API struct. Would get rid of these two. If you need them in tests, put them there locally as private helpers.